modules/Oculus_module/src/HandRetargeting.cpp

```diff
@@ -23,13 +23,24 @@ bool HandRetargeting::configure(const yarp::os::Searchable& config)
         yError() << "[HandRetargeting::configure] Empty configuration for hand retargeting.";
         return false;
     }
+    double humanHeight, robotArmSpan;
+    if (!YarpHelper::getDoubleFromSearchable(config, "humanHeight", humanHeight))
+    {
+        yError() << "[HandRetargeting::configure] Unable to find the humanHeight parameter.";
+        return false;
+    }
 
-    if (!YarpHelper::getDoubleFromSearchable(config, "scalingFactor", m_scalingFactor))
+    if (!YarpHelper::getDoubleFromSearchable(config, "robotArmSpan", robotArmSpan))
     {
-        yError() << "[HandRetargeting::configure] Unable to find the hands smoothing time";
+        yError() << "[HandRetargeting::configure] Unable to find the robotArmSpan parameter.";
```

Comment:
```suggestion
        yError() << "[HandRetargeting::configure] Unable to find the robotArmSpan parameter. It is the distance between the left and right index fingertips when the arms are aligned in T pose.";
```

Reply:
I prefer to do not mention this description in the Error, since the Error is related to non-found parameter, and the description is added in the config files.